pkg/auto/importer_test.go

```diff
@@ -25,6 +25,16 @@ import (
 	"google.golang.org/api/googleapi"
 )
 
+// Hacky, but we have to set these _before_ init() runs in the main package, and
+// variables are one of the only things initialized prior to package init
+// functions being run.
+var _ = func() interface{} {
+	retryBase = 5 * time.Millisecond
+	retryTries = 5
+	continueOnError = true
```

Comment:
Structuring the code like this might help:



```
func init(){
	continueOnErr := ....
    err := doInit()
    // determine if log.Fatal or log+continue
}


func doInit(){
   ...
   move stuff from init() to here
}
```

this way, you don't need


1. handleError(); return; pattern
2. this weird hack.
3. continueOnError GLOBAL variable.